packages/docs/docs/usage/layout-grid.md

```diff
@@ -0,0 +1,1142 @@
+can be hidden/shown based upon the value(s) of a field. It even supports the inclusion of custom render components that
+are not associated with a field. Finally, there is built-in support for doing a named look up of data within a special
+location in the `FormContext`.
+
+Since each theme that RJSF supports has its own unique grid system, the `uiSchema` that one creates for the
+`LayoutGridField` for one theme will most likely not work for any other theme unless two themes have the exact same
+grid specification props. It also requires that one understand how to use the grid system of the theme and how it
+interplays with the `LayoutGridField`'s "syntax". In other words, it will likely take some experimentation to get the
+grid looking exactly the way you want it to.
+
+Let's break down how to use the `LayoutGridField` one step at a time.
+
+## Defining the LayoutGridField to reordering rows
+
+The `LayoutGridField` is designed to be used within the `uiSchema` via the `ui:field` syntax. It must be accompanied by
+the definition of the grid within the new `ui:layoutGrid` syntax. First let's start with a fairly simple `schema` that
+looks like:
+
+```json
+{
+  "type": "object",
+  "properties": {
+    "person": {
+      "title": "Person Info",
+      "type": "object",
+      "properties": {
+        "first": {
+          "title": "First Name",
+          "minLength": 1,
+          "maxLength": 200,
+          "type": "string"
+        },
+        "middle": {
+          "title": "Middle Name",
+          "minLength": 1,
+          "maxLength": 200,
+          "type": "string"
+        },
+        "last": {
+          "title": "Last Name",
+          "minLength": 1,
+          "maxLength": 200,
+          "type": "string"
+        }
+      },
+      "required": ["first", "last"]
+    }
+  }
+}
+```
+
+If you were to pass this `schema` to the RJSF `Form` you will end up with the following UI, shown here as a simple ASCII
+drawing:
+
+```text
+Person Info
+  First Name*
+  +------------------------------------------------------------------------------+
+  |                                                                              |
+  +------------------------------------------------------------------------------+
+
+  Middle Name
+  +------------------------------------------------------------------------------+
+  |                                                                              |
+  +------------------------------------------------------------------------------+
+
+  Last Name*
+  +------------------------------------------------------------------------------+
+  |                                                                              |
+  +------------------------------------------------------------------------------+
+```
+
+If you wanted to switch the order of the fields or remove middle name entirely, you would have to change the JSON schema.
+At least you would have to if you didn't have the `LayoutGridField`.
```

Comment:
This is not true, the uiSchema already has `ui:order` and `"ui:widget": "hidden"`. That said, I like this very simple example, so I don't want you to make it more complicated for the sake of it. Just update the doc so it doesn't suggest that it's impossible without `LayoutGridField`

Reply:
Oh wow, I forgot about those :) I'll reword